packages/contracts-bedrock/test/L1/DelayedVetoable.t.sol

```diff
@@ -160,9 +160,9 @@ contract DelayedVetoable_HandleCall_TestFail is DelayedVetoable_Init {
         assumeNoClash(data);
         vm.prank(initiator);
         (bool success,) = address(delayedVetoable).call(data);
-        success;
+        assertTrue(success);
 
-        vm.expectRevert();
+        vm.expectRevert(DelayedVetoable.ForwardingEarly.selector);
```

Comment:
I actually made the opposite diff recently. See the [rationale here](https://github.com/foundry-rs/book/pull/1091).

OTOH, I can see the argument that even though the data is unused, at some point foundry may get an update to use it, so I won't block, but want to make sure we're aware that semantically this doesn't make a difference.

Reply:
From my test, it did fail if I had the wrong data in there:

<img width="1051" alt="image" src="https://github.com/ethereum-optimism/optimism/assets/17163988/1b9c1b2f-24d5-4b91-b1da-799c3affbb78">

From the logs you can see it does appear to mangle the return data, so I assumed forge held the original revert data for the comparison.

But IMO if `expectRevert` does mangle return data such that you could use any revert message, that's an argument for not using expectRevert at all for low-level calls, and instead doing

```solidity
bytes memory expectedRevertData = bytes.concat(DelayedVetoable.ForwardingEarly.selector);
(bool success, bytes memory returnData) = address(delayedVetoable).call(data);
assertFalse(success);
assertEq(expectedRevertData, returnData);
```

Reply:
Confirmed this should work and opened https://github.com/foundry-rs/book/pull/1112 to revert your book PR :) 

If you're seeing otherwise though let me know and we can debug

Reply:
OK, SGTM